app/components/tittle_component.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class TittleComponent < ViewComponent::Base
```

Comment:
`Tittle`?

Reply:
Also, this is something we would like to do with a utility class instead of a component